_data/bhaskargupta98/bhaskargupta98.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Bhaskar Gupta # No longer than 18 characters
+institution: Thapar Institute of Engg and Tech 🚩 # no longer than 58 characters
+profile_pic: bhaskar.jpeg # Name and extension of your profile picture(ex. mona.png)
```

Comment:
Small detail - you may want to rename the image to the ```.jpg``` file extension, in case ```.jpeg``` may cause some issues.